MPSock.java

```diff
@@ -219,15 +219,8 @@ public TCPSendSock addSendSocket(ConnID cID, TCPSendSock sendSock) {
     public TCPReceiveSock createEstSocket(ConnID cID) {
         // TODO: extend to multiple IP addresses
         // need to find the next available port
-        ConnID reversecID = cID.reverse();
```

Comment:
We need to keep this reversal. This is used because the MPSock.estMap sees cIDs from it's perspective.

Later, the constructed newcID is also a reversal